READMEv2.md

```diff
@@ -0,0 +1,134 @@
+<h1 align="center"> OADP Operator v2 </h1>
+
+<p align="center">  OADP is OpenShift API for Data Protection operator. This operator sets up and 
+installs <a href="https://velero.io/">Velero</a> on the OpenShift platform. </p>
+
+<p align="center"><img src="https://img.shields.io/badge/Verison-2.0-yellow"/>
+<img src="https://img.shields.io/github/go-mod/go-version/openshift/oadp-operator"/>
+<img src="https://img.shields.io/badge/license-Apache_0.2-red" alt="license" />
```

Comment:
Our license is 2.0

Reply:
@kaovilai I'll just remove all badges, since yours are nicer